scripts/update-issues.ts

```diff
@@ -144,21 +211,23 @@ async function update() {
   for (const [id, data] of Object.entries(features)) {
     const dates: string[] = [];
     for (const [browser, version] of Object.entries(data.status.support)) {
-      const v = version.replace('≤', '');
-      const release = browsers[browser].releases.find((r) => r.version === v);
+      const v = version.replace("≤", "");
+      const release = browsers[browser as keyof typeof browsers].releases.find(
+        (r) => r.version === v,
+      );
       if (release) {
         dates.push(release.date);
       }
     }
     // Add a date-like string that will sort after any real date as a tiebreaker
     // when N dates are the same and one feature has more than N dates. This
     // also ensures that features that aren't shipped sort last.
-    dates.push('9999-99-99');
+    dates.push("9999-99-99");
     dates.sort();
-    sortKeys.set(id, dates.join('+'));
+    sortKeys.set(id, dates.join("+"));
   }
   const sortedIds = Object.keys(features).sort((a, b) => {
-    return sortKeys.get(a).localeCompare(sortKeys.get(b));
+    return sortKeys.get(a)!.localeCompare(sortKeys.get(b)!);
```

Comment:
Ah, `!` is useful, I did see an TypeScript warning here but didn't know how to silence it, or didn't want to make the code uglier to satisfy TS.